docs/NPC_Swap_Tutorial.md

```diff
@@ -0,0 +1,224 @@
+
+Since Caruo's outfit is in `chunk1` but our mod will be in `chunk0`, we will also need to extract the depends (which are the assets the outfit uses) and `patch` them to chunk0. Remember that while chunk1 can access chunk0, chunk0 can not access chunk1. If we try to run our mod without including the depends it needs, the game will simply crash.
+
+While you still have the correct file selected in the left box, left-click `Extract Hash Depends` in the right box, then select `Extract Recursive Hash Depends` and extract them to the same folder as the JSON. 
+
+The output folder should now have a `.entity.json` file and a folder named `ALLDEPENDS`. Inside this folder, you'll find a `00873434CB4F9FCD.TEMP` folder, and then inside that folder, you'll have various `chunkX` folders. 
+
+When creating suit swaps or modifications, we'll ultimately want to place our mod in `chunk0` for two reasons: 1) it's a 'universal' chunk that is read from everywhere, and 2) it contains all of 47's selectable suits. Therefore, we won't need the depends from chunk0 or its patches, so we can go ahead and delete the `chunk0`, `chunk0patch1`, and `chunk0patch2` folders.
+
+That will leave us with 3 folders: `chunk1`, `chunk1patch2`, and `chunk8`. First, it's important to mention that outfits can only read from the same chunk they reside in. Therefore, if we're swapping an outfit that's located in chunk1, we will only ever need the depends from chunk1 and its patches. So, we can safely delete `chunk8`. Next, we'd usually start by transferring the depends from the patchX chunks to their base chunks. For example, we'd move the depends from `chunk1patch2` to `chunk1`, and replace anything if it asks. However, in this case, we can see `chunk1patch2` only contains a TBLU file and a TEMP file. If we were to restore `chunk8`, we would see that it contains the same TEMP/TBLU pair, which matches what we saw in RPKG Tool. Namely, that `chunk1patch2` and `chunk8` contained duplicate TEMP/TBLU pairs. We also know from earlier that TBLU and TEMP files contain outfit templates, but we've already converted Caruso's TEMP/TBLU pair to a QuickEntity JSON, so we no longer need them. Therefore, that only leaves `chunk1`.
+
+If we examine the contents of `chunk1`, we can see it contains 5 different folders with various file types. If we check the [Glacier 2 Engine file formats](https://wiki.notex.app/glacier2/fileformats), we can see this chunk appears to include important assets, like meshes (PRIM), animation bone data (BORG), textures (TEXT), etc. So, these must be the depends we're looking for. We'll want to move these 5 folders (BORG - TEXT) to the same location as the `00873434CB4F9FCD.entity.json` file we extracted. The chunkX.meta files can be ignored. Once this is done, you can delete the `ALLDEPENDS` folder and its remaining content.

```

Comment:
> animation bone data

Change this to `bone rig`, since it's what it is.

> textures (TEXT)

Change this to `textures (TEXT/D)` or `textures (TEXT/TEXD)`